### PR TITLE
Python static analysis

### PR DESCRIPTION
**Pull request by vbmade2000**
Formatted python code according to pycodestyle

### CODE REVIEW

**Comment by vbmade2000** [comment]
@brendandburns Can you please review and merge this PR ?

File: python/setup.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
     platforms='linux',
     keywords=['kubernetes', 'docker', 'container', 'metaparticle'],
     # latest from https://pypi.python.org/pypi?%3Aaction=list_classifiers
-    classifiers = [
+    classifierssss = [
```
Comment (brendandburns) [inline python/setup.py]: This is weird, why the extra sss?

**Comment by brendandburns** [comment]
Generally looks good except for one comment.

File: python/setup.py (hunk 1 above)
Comment (vbmade2000) [reply python/setup.py]: Oh. Must be a typing issue. I'll correct it.

**Comment by vbmade2000** [comment]
@brendandburns Can you recheck ?

**Comment by brendandburns** [comment]
LGTM.

**Pull request merged**